Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/cluster/ClusterDetail.java

```diff
@@ -273,6 +273,60 @@ public void getConfigurationInfo() throws IOException, HDIException, AzureCmdExc
         }
     }
 
+    @Nullable
+    public String getDefaultStorageRootPath() {
+        if (!(clusterOperation instanceof ClusterOperationNewAPIImpl)) {
+            return null;
+        }
+
+        try {
+            Map<String, String> coresiteMap = ((ClusterOperationNewAPIImpl) clusterOperation).getClusterCoreSiteRequest(clusterRawInfo.getId())
+                    .toBlocking()
+                    .singleOrDefault(null);
+
+            if (coresiteMap == null) {
+                throw new HDIException("Error getting cluster core-site");
+            }
+
+            String containerAddress = null;
+            if (coresiteMap.containsKey(DefaultFS)) {
+                containerAddress = coresiteMap.get(DefaultFS);
+            } else if (coresiteMap.containsKey(FSDefaultName)) {
+                containerAddress = coresiteMap.get(FSDefaultName);
+            }
+            log().info("fs.defaultFS: " + containerAddress);
+
+            if (containerAddress == null) {
+                throw new HDIException("Failed to get default storage account");
```

Comment:
May log is enough, returning `null` for UI to use interactive session.

Reply:
Will apply the change in the next iteration

---
In reply to: [278013850](https://github.com/Microsoft/azure-tools-for-java/pull/3075#discussion_r278013850) [](ancestors = 278013850)